src/zc/buildout/easy_install.py

```diff
@@ -1619,25 +1622,19 @@ def unpack_egg(location, dest):
     setuptools.archive_util.unpack_archive(location, dest)
 
 
-WHEEL_TO_EGG_WARNING = """
-Using unpack_wheel() shim over the deprecated wheel_to_egg() hook.
-Please update your wheel extension implementation for one that installs a .whl
-handler in %s.UNPACKERS
-""".strip() % (__name__,)
+WHEEL_WARNING = """
+*.whl file detected (%s), you'll need setuptools > 38.2.3 for that
+or an extension like buildout.wheel
```

Comment:
Can you please make it?:

    or an extension like buildout.wheel > 0.2.0

Reply:
Good idea. Done!

Reply:
Should these be `>` or `>=`?

Reply:
Definitely `>= 38.2.3` for setuptools, good catch!

Reply:
Whoopsie! Fixed in f76788c06b7652cca7213044eb0e142e0612fd57 :-)